docs/draft/webview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: Dapp browser API usage
+version: 0.1.0
+status: Draft
+authors: 
+---
```

Comment:
Missing permalink and number, see other specs for format

Reply:
Added permalink, but what should the number be?

Reply:
Depends on what order PRs are in :sweat_smile: I think 10 might be free? Check existing PRs, we can always change it at end before merge

Reply:
Modified permalink to include number.